.gitignore

```diff
@@ -2,3 +2,7 @@
 *.env
 /Welolo/node_modules/*
 /Welolo/client/node_modules/*
+<<<<<<< HEAD
+=======
+*.DS_Store
```

Comment:
One of these made it in.

Reply:
One other issue that I noticed was that the emergepay modal will open even when fields are empty